src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery extends ColorSupplier {
+    public Ball getRandomBall() {
+
+        Random random = new Random();
+        int randomNumber = random.nextInt(10);

```

Comment:
**All magic numbers in your code should be constants.**
[Common mistakes](https://mate-academy.github.io/jv-program-common-mistakes/java-core/oop/lottery#all-magic-numbers-in-your-code-should-be-constants)